site/content/docs/08-link-options.md

```diff
@@ -19,4 +18,16 @@ We can mitigate that by *prefetching* the data. Adding a `rel=prefetch` attribut
 
 > `rel=prefetch` is a Sapper idiom, not a standard attribute for `<a>` elements
 
-<!-- TODO add a function to prefetch programmatically -->
\ No newline at end of file
+<!-- TODO add a function to prefetch programmatically -->
+
+### rel=external
+
+The sapper router listens for clicks on `<a>` elements and bypasses the normal browser navigation for relative (same origin) paths. We sometimes need links that are targetted at routes that are not part of the sapper app and hence should be reached by a normal browser navigation.
```

Comment:
nit: I would use "targeted" instead of "targetted". I'm not sure the latter is necessarily incorrect, but it's very uncommon: https://english.stackexchange.com/questions/205815/is-targetted-a-standard-british-english-spelling

Reply:
Done, thanks.